### ISSUE TITLE
Avoid window 'You can now close this window'

### ISSUE THREAD

**Issue by Bat-m** [appsup-dart/openid_client#27]
Hello

When the login is correct and openid authorize
we can see a window 
'You can now close this window'

is it possible to avoid this part ?

**Comment by edihasaj** [comment]
Is there any way out to fix this ?

**Comment by edihasaj** [comment]
I found a workaround on this. I modified the openid_client.io file in code line 60 at this time

```
request.response.headers.set('Content-type', 'text/html');
request.response.writeln('<html>'
   //'<h1>You can now close this window</h1>' // how it was
   '<h1>Redirecting back...</h1>' // now
   '<script>window.close();</script>'
   '</html>');
await request.response.close();
```

**Comment by indiealexh** [comment]
Is there any reason content needs loaded here? I just removed the response stuff and it closes fine.

**Comment by SylvainAssemat** [comment]
@edihasaj 

Thanks for the fix , i will do the same.

**Comment by rbellens** [comment]
Should be fixed with merging pull request #66. Will be released soon.
